tests/foreman/upgrade/test_organization.py

```diff
@@ -0,0 +1,96 @@
+# -*- encoding: utf-8 -*-
+# pylint: disable=no-self-use
```

Comment:
NACK, no-self-use is a good indicator that some method can be a plain function. Move the method outside the class.